xapian-applications/omega/worker_comms.cc

```diff
@@ -0,0 +1,104 @@
+/** @file worker_comms.cc
+ * @brief Communication with worker processes
+ */
+/* Copyright (C) 2011 Olly Betts
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+
```

Comment:
I am not sure about what to put in the license comments.

Reply:
If you've made significant changes add a "Copyright (C) 2019 Bruno Baruffaldi" after the existing "Copyright ..." lines.

The exact criteria for copyrightability are a bit complex, but the work you're doing during GSoC should definitely count.